operator/pkg/name/name.go

```diff
@@ -72,6 +72,22 @@ var (
 		CNIComponentName,
 	}
 	allComponentNamesMap = make(map[ComponentName]bool)
+
+	// ComponentNameToHelmComponentPath defines mapping from component name to helm component root path.
+	// TODO: merge this with the componentMaps defined in translateConfig
+	ComponentNameToHelmComponentPath = map[ComponentName]string{
+		PilotComponentName:           "pilot",
+		GalleyComponentName:          "galley",
+		SidecarInjectorComponentName: "sidecarInjectorWebhook",
+		PolicyComponentName:          "mixer.policy",
+		TelemetryComponentName:       "mixer.telemetry",
+		CitadelComponentName:         "security",
+		CertManagerComponentName:     "certmanager",
+		NodeAgentComponentName:       "nodeagent",
+		IngressComponentName:         "gateways.istio-ingressgateway",
+		EgressComponentName:          "gateways.istio-egressgateway",
+		CNIComponentName:             "cni",
+	}
```

Comment:
We need prometheus, grafana, etc as well.

Reply:
Any update on this?

Reply:
I'm adding those in my PR: https://github.com/istio/istio/pull/20352